modules/base/lightsource/cameralightsource.cpp

```diff
@@ -37,9 +37,12 @@ namespace {
         openspace::properties::Property::Visibility::NoviceUser
     };
 
+    // This `LightSource` type represents a light source placed at the position of the
+    // camera. An object with this light source will always be illuminated from the
+    // current view direction.
     struct [[codegen::Dictionary(CameraLightSource)]] Parameters {
         // [[codegen::verbatim(IntensityInfo.description)]]
-        std::optional<float> intensity;
+        std::optional<float> intensity [[codegen::inrange(0.0, 1.0)]];
```

Comment:
Is this correct?  Looks like we can theoretically specify an intensity larger than 1

Reply:
I don't think we need to limit the intensitiy to `[0,1]`.  In some examples I've used light intensities >1 to make models brighter.   And I could imagine values <0 for dramatic effect (to achieve an inverse diffuse lighting, for example)

Reply:
Removing this change then :)